src/devices/uFire_ISE/samples/uFire_ISE.Sample.cs

```diff
@@ -12,8 +12,9 @@ namespace Iot.Device.UFire.Sample
     /// </summary>
     public static class Program
     {
-        private const int BUSID = 1;
-        private const int I2CADDRESS = 0x3F;
+        private const int BusId = 1;
+        // 0x3F is the default address of all sensors
+        private const int I2cAddress = 0x3F;
```

Comment:
So if this is the default address of all sensors, then it would make sense to expose this const in the main base UFireIse class as public, and also consider changing for byte, so something like this:
```csharp
public class UFireIse : IDisposable
{
    /// <summary>
    /// 0x3F is the default address of all sensors
    /// </summary>
    public const byte I2cAddress = 0x3F;
}
```